acceptance/testdata/builder/container/layers/different_layer_dir_from_env/group.toml

```diff
@@ -0,0 +1,11 @@
+[[group]]
```

Comment:
I don't think that you need the `.gitkeep` in this folder because it contains other files (this file and `plan.toml`)

Reply:
deleted